ehr2vec/configs/example_configs/04_finetune.yaml

```
@@ -14,7 +14,8 @@ paths:
   # tokenized_dir:"tokenized"
   tokenized_file: "tokenized_finetune.pt" # can also be a list
   tokenized_pids: "pids_finetune.pt" # can also be a list
-  predefined_splits: "outputs/example_pretraining/example/finetune_EXAMPLE_CENSOR_censored_24_hours_pre_EXAMPLE_CENSOR_followup_start_at_index_date_run_test"
+  
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)